src/libs/PolicyUtils.ts

```diff
@@ -528,6 +528,17 @@ function getNetSuiteTaxAccountOptions(policy: Policy | undefined, subsidiaryCoun
         }));
 }
 
+function isSyncTaxEnabled(policy: Policy | undefined) {
+    const netSuiteConfg = policy?.connections?.netsuite?.options?.config;
+    const {subsidiaryList} = policy?.connections?.netsuite?.options?.data ?? {};
+    if (!netSuiteConfg || !subsidiaryList) {
+        return false;
+    }
+
+    const selectedSubsidiary = (subsidiaryList ?? []).find((subsidiary) => subsidiary.internalID === netSuiteConfg?.subsidiaryID);
+    return !!netSuiteConfg?.suiteTaxEnabled && !isUSASubsidiaryNetSuite(selectedSubsidiary?.country);
```

Comment:
> The Tax option will only present itself when non-US subsidiaries are selected, and we’ve detected that Legacy NetSuite Tax or  SuiteTax is enabled on that subsidiary in NetSuite.

Still clarifying the condition for `legacy netsuite or suite tax`.

Reply:
got it clarified with @yuwenmemon. Just checking subsidiary countries is enough.

Reply:
Updated the condition.